components/serializers/fixes.hpp

```diff
@@ -0,0 +1,8 @@
+#undef REGISTER_OBJECT_WRAPPER
```

Comment:
This macro had several problems in my opinion.  The biggest is that the MyClass typedef can't be undone, literally preventing there from being more than one serializer per file. :-(  Apparently in OSG they worked around this by having one serializer source file per class. :-(